Gruntfile.js

```diff
@@ -652,17 +669,36 @@ module.exports = function (grunt) {
             // Remove /stylus from the path, since dist/assets won't have it
             expectedFilenames.push(cssFilename.replace("/stylus", ""), minifiedCSSFilename.replace("/stylus", ""));
         });
+        verifyFilesTaskFunc(message, expectedFilenames, "dist/assets");
+    });
 
-        var report = verifyFiles("dist/assets", expectedFilenames);
-
-        displayVerifyFilesReport(report);
-        processVerifyFilesReport(report);
+    grunt.registerTask("verifyDistFonts", "Verifies that the expected /dist/ font files were created", function () {
+        var expectedFonts = {
+            openDyslexic: {
+                cwd: "src/lib/open-dyslexic/fonts",
+                pattern: "*.woff",
+                message: "Verify opendyslexic font files are present in /dist/ directory",
+                distDirectory: "dist/assets/src/lib/open-dyslexic/fonts"
+            },
+            openSans: {
+                cwd: "src/lib/opensans/fonts",
+                pattern: "*.woff",
+                message: "Verify opensans font files are present in /dist/ directory",
+                distDirectory: "dist/assets/src/lib/opensans/fonts"
+            }
+        };
```

Comment:
Looking at this, seems a bit odd that the configuration for which files to check isn't in the typical grunt config. I think ideally that would be the case for all the verification steps. Is there a reason why the configuration is hardcoded in each task?

Reply:
Is your suggestion simply to move the functions etc that generate the file lists into the Grunt config, or to move to manually maintaining the list of files to verify? (I'm unclear as to which)

Reply:
I mean would it be possible to make use of grunts own [files](https://gruntjs.com/configuring-tasks#files) mechanics.

Reply:
I've finished initial refactoring to use Grunt's file mechanics where possible. Where not (the generated JS files), I've made the multitask accept a custom `verifyFilesListFunc`. 

So task configuration either uses the standard Grunt file mechanics where there's a one-to-one mapping (possibly with renaming) between source files and destination files, or specifies a function to return an array of files to check. I think this should have the flexibility required for future additions - most of them will fall into the first case, but the second is available if the verification logic is not as straightforward as "these source files are compiled into these destination files".

Let me know your thoughts.